lib/modules/collapseWhitespace.es6

```diff
@@ -35,20 +67,43 @@ export default function collapseWhitespace(tree, options, collapseType, tag) {
 }
 
 
-function collapseRedundantWhitespaces(text, collapseType, isTopLevel = false) {
+function collapseRedundantWhitespaces(text, collapseType, shouldTrim = false, currentTag, prevNodeTag, nextNodeTag) {
     if (!text || text.length === 0) {
         return NONE;
     }
 
     if (collapseType === 'aggressive') {
-        text = text.replace(indentPattern, NONE);
+        text = text.replace(indentPattern, SINGLE_SPACE);
```

Comment:
Shouldn't remove all indents since there might be a `noTrimWhitespacesArroundElement` nearby.